Dockerfile

```diff
@@ -20,24 +20,26 @@ RUN apt-get update -q && \
         git \
         curl \
         wget \
-        curl \
         software-properties-common \
     && \
     locale-gen en_US.UTF-8 && export LANG=en_US.UTF-8 && \
     add-apt-repository ppa:git-core/ppa -y && \
     add-apt-repository ppa:ondrej/php -y && \
-    echo 'deb http://apt.newrelic.com/debian/ newrelic non-free' | sudo tee /etc/apt/sources.list.d/newrelic.list && \
```

Comment:
Oh wow I didn't know that ... that's actually super helpful for my `scarlettOS` project haha